backend/lib/openfisca/mapping/index.js

```diff
@@ -212,6 +212,14 @@ function applyHeuristicsAndFix(testCase, sourceSituation) {
 }
 exports.applyHeuristicsAndFix = applyHeuristicsAndFix
 
+function filterRequestedVariablesBySituation(requestedVariables, situation) {
+  const variables = { ...requestedVariables }
+  if (situation.menage.depcom?.startsWith("28")) {
```

Comment:
Même si ça rajoute de la verbosité est-ce que ça serait pas mieux de faire quelque chose comme ça :
```suggestion
  const excludeDepcom = ["28"]
  if(excludeDepcom.includes(situation.menage.depcom?.substring(0, 2))) {
```

Ça permettrait sans avoir le contexte de plus facilement comprendre qu'on exclus le `28` tout en permettant de rajouter des départements dans le futur.

Si tu trouves que ça rajoute trop de complexité dans ce cas je mettrais bien un commentaire type
```
// exclusion du département 28 parce que blah blah blah
```

Reply:
En effet je suis d'accord qu'il faut expliquer un peu plus le contexte. Je serais d'avis d'ajouter un commentaire
```
// exclusion du département 28 parce qu'elles sont mieux simulées par les variables `eure_et_loir_eligibilite_*`
```